low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -539,6 +541,56 @@ def process(self):
         logger.info(f"{self.name} interface not implemented.")
 
 
+class PreUpgradeCmd(Cmd):
+    """PreUpgrade support for SSPL componenet."""
+
+    name = "pre_upgrade"
+
+    def __init__(self, args):
+        super().__init__(args)
+
+    def validate(self):
+        # Common validator classes to check Cortx/system wide validator
+        pass
+
+    def process(self):
+        logger.info(f"{self.name} interface not implemented.")
```

Comment:
Instead mention
logger.info(f"Nothing to be done for '{self.name}' ")

Reply:
Done